ibis/backends/omniscidb/compiler.py

```diff
@@ -2,10 +2,10 @@
 from io import StringIO
 
 import ibis
+import ibis.backends.base_sqlalchemy.compiler as compiles
```

Comment:
Hi @jreback, yes I have changed this also because now there is already a
`compiles = OmniSciDBExprTranslator.compiles` in line 296.

Have changed all the type hints and the docstrings too. please let me know if it looks ok now